packages/editor/src/components/page-attributes/parent.js

```diff
@@ -212,8 +212,10 @@ function PostParentToggle( { isOpen, onClick } ) {
 			className="editor-post-parent__panel-toggle"
 			variant="tertiary"
 			aria-expanded={ isOpen }
-			// translators: %s: Current post parent.
-			aria-label={ sprintf( __( 'Change parent: %s' ), parentTitle ) }
+			aria-label={
+				// translators: %s: Current post parent.
```

Comment:
Maybe??? 

In other locations you've changed one line translator comments to use this format, is that required here too?

```suggestion
				/* translators: %s: Current post parent. */
```

Reply:
The comment style doesn't really matter. When I changed it, it was probably because of copy/paste.